tools/releasing/deploy_staging_jars.sh

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 #
 
+## check required Java and Maven version
+. check_java_maven_versions.sh
```

Comment:
Here, we don't have any other commands happening before the Maven call. Therefore, we would get the version check before anything else happens as well.